components/resume/job/job.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import { Job } from '../types';
+import Position from './position';
+
+const Job: FC<Job> = ({ companyName, positions, highlights }) => {
```

Comment:
For now, will make these prop types the same as the `Job` type, but in the future will convert this to a component specific prop type if it expands. ie

```
interface JobProps extends Job {
  newProp: type
}
```